docs/user/03-11-customize_secret.md

```diff
@@ -1,5 +1,8 @@
 # Customize the Default Credentials and Access
 
+> [!NOTE]
```

Comment:
we shouldn't document features that don't exist yet. 
https://wiki.one.int.sap/wiki/display/PSUA/UA-030

I understand that probably no one will care about following the product standard for our kyma-project.io docs, but I'm not sure what's good practice here.

Reply:
I already explained. Let's try this way for a while.